k8s/spicedb.yaml

```diff
@@ -26,56 +33,116 @@ spec:
       port: 50051
       protocol: "TCP"
       targetPort: 50051
-    - name: "internal"
+    - name: "http"
+      port: 8443
```

Comment:
Do we need this port?

Reply:
Good question. To be honest, the spicedb config probably needs to be looked at more carefully. I was just trying to get something working here.

I am also now wondering if we should mirror the prod DeploymentConfig spec here somehow. Can we (with little effort) using the prod template for local dev for example? (ImageStreams/triggers would need to be replaced by make targets.)

Reply:
Just checking what we have in prod, and we have http/8443 there too. I'll keep it here for now, and we can remove from both in the future if want to.